libursa/DatabaseConfig.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <map>
+#include <string>
+#include <string_view>
+
+#include "Json.h"
+
+class ConfigKey {
```

Comment:
What's the point of `ConfigKey` class, if it's only a wrapper on strings :thinking: